pkg/webhook/core.oam.dev/v1alpha2/applicationconfiguration/validating_handler.go

```diff
@@ -82,6 +82,12 @@ func (h *ValidatingHandler) Handle(ctx context.Context, req admission.Request) a
 		if err != nil {
 			return admission.Errored(http.StatusBadRequest, err)
 		}
+		if !obj.ObjectMeta.DeletionTimestamp.IsZero() {
+			// skip validating the AppConfig being deleted
```

Comment:
Why? When `vela delete` an application, it will delete Components before deleting AppConfig. But validating-webhook for AppConfig will fail for "Component Not Found". That validation failure will block updating AppConfig finalizers. Consequently, it cannot delete the appConfig.

Reply:
Why delete appconfig will trigger webhook? We only define `create` and `update` for this.

Reply:
After handling finalizers, it has to update AppConfig.

Reply:
I see, then it's fine.